.github/workflows/CI.yml

```diff
@@ -47,16 +47,16 @@ jobs:
             SDK: portal2
             spt-filename: spt-p2
     steps:
-    - name: Checkout AlliedModders SDK
-      uses: actions/checkout@v2
-      with:
-        repository: alliedmodders/hl2sdk
-        ref: ${{ matrix.SDK }}
     - name: Checkout SPT
       uses: actions/checkout@v2
       with:
         path: utils\SourcePauseTool
-        submodules: recursive
+    - name: Checkout SPTLib
+      run: git submodule update --init SPTLib
+      working-directory: utils\SourcePauseTool
+    - name: Checkout SDK
+      run: git submodule update --init SDK\${{ matrix.SDK }}
+      working-directory: utils\SourcePauseTool
```

Comment:
Check out manually so that it doesn't download the repo 5 times, right?

Reply:
yes, it only checks out the relevant submodule for the configuration